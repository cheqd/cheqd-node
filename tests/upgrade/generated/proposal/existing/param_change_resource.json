{
    "title": "Resource Param Change",
    "description": "This is a simple example of a param change - double the values",
    "changes": [
        {
            "subspace": "resource",
            "key": "feeparams",
            "value": "{\"image\": {\"denom\": \"ncheq\", \"amount\": \"10000000000\"}, \"json\": {\"denom\": \"ncheq\", \"amount\": \"4000000000\"}, \"default\": {\"denom\": \"ncheq\", \"amount\": \"2000000000\"}, \"burn_factor\": \"0.600000000000000000\"}"
        }
    ],
    "deposit": "10000000000ncheq"
}