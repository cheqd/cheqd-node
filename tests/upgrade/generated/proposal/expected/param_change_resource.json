{
    "image": {
        "denom": "ncheq",
        "amount": "20000000000"
    },
    "json": {
        "denom": "ncheq",
        "amount": "5000000000"
    },
    "default": {
        "denom": "ncheq",
        "amount": "10000000000"
    },
    "burn_factor": "0.600000000000000000"
}